config.go

```diff
@@ -187,6 +192,16 @@ func parseMetrics(conf map[interface{}]interface{}) map[int][]keyExtract {
 			} else {
 				multiply = 1
 			}
+			if divide_key, ok := m["divide"]; ok == true {
+				divide = divide_key.(int)
```

Comment:
A check for 0 perhaps?

Reply:
It looked like there was a check for `>1` in [datapool.go](https://github.com/mathpl/logmetrics/pull/5/files#diff-17741e5640babdaf5c2c5668615dc8b5R153).  I guess we can make it more explicit and fail with an error at config time?

I was just wondering about dumping "divide" and changing "multiply" to be a float instead - what do you think?

Reply:
Obviously it'd work but we'd get float arithmetic on multiply as well. Better keep it to integer if we can. I'd rather have a check at config time, that's going to be easier to diagnose. Add that and we're good to go.

Reply:
Added checks+Fatalf's to config.go.